setup.py

```diff
@@ -35,6 +35,10 @@ def find_version():
           'argparse_addons',
           'typing_extensions>=3.10.0.0',
           'crccheck',
+          'matplotlib',
+          'parameterized',
+          'codespell',
+          'nala',
```

Comment:
These are test dependencies, not runtime dependencies. They must not be included in `install_requires`

Reply:
where do i put them, I guess the question is still where is the requirements.txt